### ISSUE TITLE
Firmware 18.0.1 -- Compatibility

### ISSUE THREAD

**Issue by RHX0R3** [suchmememanyskill/TegraExplorer#95]
Hello dear developer and TE community,

I would like to know when the currently released firmware `18.0.1` will be officially supported?

According to the patch notes, together with version `18.0.0`, this seems to be more of an update for system stability and improvement of the WiFi function.

Will it take a while until full compatibility can be achieved?

I would be very happy to receive a feedback.

Best regards,
X0R3

**Comment by suchmememanyskill** [comment]
Is anything currently broken then?

Firmware update compatibility is only for keydump (on erista, keydump on mariko is unsupported)

If keydump works, TE does not need an update

**Comment by RHX0R3** [comment]
Hello,

I can't say that, currently.

I was asking beforehand, because I am quite new into the whole modding thematic.

I prepared myself with a PicoFly Modchip for my v2 Switch. And I followed the guide on how to set things up, after the successful soldering.
Maybe I got mislead or something; sorry if that is a inconvenience.
Just wanted to make sure, I start with everything compatible software-wise.

Best regards,
X0R3
